rulewerk-owlapi/src/main/java/org/semanticweb/rulewerk/owlapi/OwlToRulesConversionHelper.java

```diff
@@ -97,19 +98,23 @@ public static Predicate getConjunctionAuxiliaryClassPredicate(final Collection<O
 	}
 
 	static String getAuxiliaryClasNameConjuncts(final Collection<OWLClassExpression> conjuncts) {
-		return getAuxiliaryClassName("aux-conjunction", conjuncts);
+		return getAuxiliaryOWLObjectName("aux-class-conjunction", conjuncts);
 	}
 
 	static String getAuxiliaryClassNameDisjuncts(final Collection<OWLClassExpression> disjuncts) {
-		return getAuxiliaryClassName("aux-disjunction", disjuncts);
+		return getAuxiliaryOWLObjectName("aux-class-disjunction", disjuncts);
+	}
+	
+	static String getAuxiliaryPropertyNameDisjuncts(final Collection<OWLObjectPropertyExpression> disjuncts) {
+		return getAuxiliaryOWLObjectName("aux-objectPropery-disjunction", disjuncts);
 	}
 
-	private static String getAuxiliaryClassName(final String prefix,
-			final Collection<OWLClassExpression> owlClassExpressions) {
+	private static String getAuxiliaryOWLObjectName(final String prefix,
+			final Collection<? extends OWLObject> owlObjects) {
 		final MessageDigest messageDigest;
 		try {
 			messageDigest = MessageDigest.getInstance("MD5");
-			for (final OWLClassExpression owlClassExpression : owlClassExpressions) {
+			for (final OWLObject owlClassExpression : owlObjects) {
```

Comment:
Since this is no longer necessarily a class expression, the variable name should be changed as well.

Reply:
done.